jest.config.js

```diff
@@ -0,0 +1,6 @@
+// jest.config.js

```

Comment:
This file is not necessary, the tests inside the package should be executed with `bun test` https://bun.sh/docs/cli/test